x/plans/proposal_handler.go

```diff
@@ -29,6 +29,17 @@ func handlePlansProposal(ctx sdk.Context, k keeper.Keeper, p *types.PlansAddProp
 	// add the plans to the plan storage
 	for _, planElem := range p.Plans {
 		logger := k.Logger(ctx)
+		if len(planElem.PlanPolicy.GetChainPolicies()) > 0 {
+			err := k.ValidateChainPolicies(ctx, planElem.GetPlanPolicy())
```

Comment:
we dont need this, lets not verify with the specs

Reply:
done